processing/data_collection/gazette/spiders/ce_fortaleza.py

```diff
@@ -0,0 +1,76 @@
+import re
+import dateparser
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+
+PDF_URL = 'http://apps.fortaleza.ce.gov.br/diariooficial/{}'
+
+def generate_urls_by_year():
```

Comment:
Don't create a function to get the list of initial URLs, redefine `start_requests` method of `scrapy.Spider` class . It is the right way to do it. See https://github.com/okfn-brasil/diario-oficial/pull/40 for more information.